README.md

```diff
@@ -1,87 +1,235 @@
-# Unified-Language-Server
+# unified-language-server
+
+[![Build][build-badge]][build]
+[![Coverage][coverage-badge]][coverage]
+[![Downloads][downloads-badge]][downloads]
+[![Size][size-badge]][size]
+[![Sponsors][sponsors-badge]][collective]
+[![Backers][backers-badge]][collective]
+[![Chat][chat-badge]][chat]
+
+Create a **[language server][]** based on **[unified][]** ecosystems.
+
+## Contents
+
+*   [What is this?](#what-is-this)
+*   [When should I use this?](#when-should-i-use-this)
+*   [Install](#install)
+*   [Use](#use)
+*   [API](#api)
+    *   [`createUnifiedLanguageServer(options)`](#createunifiedlanguageserveroptions)
+*   [Examples](#examples)
+*   [Types](#types)
+*   [Compatibility](#compatibility)
+*   [Related](#related)
+*   [Contribute](#contribute)
+*   [License](#license)
+
+## What is this?
+
+This package exports a function which can be used to create a
+[language server][] based on [unified][] processors.
+It can do the following:
+
+*   format documents based on a unified processor
+*   validate documents based on a unified processor
+*   support configuration files (such as `.remarkrc`) using [unified-engine][]
```

Comment:
question on scope, is there interest in having this also be able to generate previews of rendered documents inside vscode? (for example `remark-parse`, with `remark-rehype`, and `rehype-stringify`)

Reply:
Possibly. That’s not just a simple editor feature though. I think this would require a way to implement custom command, then custom logic in the client (`vscode-remark`) to implement this.

So this might be interesting for later.